packages/neuron-wallet/src/addresses/entities/address.ts

```diff
@@ -49,6 +49,12 @@ export default class Address extends BaseEntity {
   })
   version!: AddressVersion
 
+  @Column({
+    type: 'text',
```

Comment:
Does this description will be long ? Should I use `text` or `varchar` ? @ashchan

Reply:
Limit to a max length is reasonable.

Reply:
`varchar` is limited to 255 chars and text is much longer, so should we change to varchar to have a limit ?

Reply:
I believe `varchar` is enough. On UI or app layer, we can limit that to 100 UTF8 characters.

Reply:
updated to `varchar`